docs/raspberry-set-up.md

```diff
@@ -0,0 +1,52 @@
+#AEA-Setup on a Raspberry Pi.
```

Comment:
please add the title to the mkdocs.yml in a relevant section and then remove it here